RecoPPS/ProtonReconstruction/interface/ProtonReconstructionAlgorithm.h

```diff
@@ -36,12 +35,12 @@ class ProtonReconstructionAlgorithm {
 
   /// run proton reconstruction using single-RP strategy
   reco::ForwardProton reconstructFromSingleRP(const CTPPSLocalTrackLiteRef &track,
-                                              const LHCInfo &lhcInfo,
+                                              const float &energy,
```

Comment:
Fundamental types are much faster when passed by value (when not used as output argument)
```suggestion
                                              const float energy,
```
(also elsewhere)